experimental/packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -100,7 +102,7 @@ export abstract class InstrumentationBase<T = any>
     module: InstrumentationModuleDefinition<T>,
     exports: T,
     name: string,
-    baseDir?: string
+    baseDir?: string | void
```

Comment:
`void` should be the same as `undefined` no ?

Reply:
This was adjusted to work with the new hookFn on line 172. The [type](https://github.com/DataDog/import-in-the-middle/blob/main/index.d.ts#L24) for `HookFn` from IITM is `string|void` - omitting `|void` or replacing `void` with `undefined` results in a type error `Type 'void' is not assignable to type 'string | undefined'.`